pkg/entities/types.go

```diff
@@ -7988,6 +7988,8 @@ type DashboardVariable struct {
 	NRQLQuery *DashboardVariableNRQLQuery `json:"nrqlQuery,omitempty"`
 	// Variable identifier.
 	Name string `json:"name,omitempty"`
+	// Options applied to the variable
+	Options DashboardVariableOptionsInput `json:"options,omitempty"`
```

Comment:
```suggestion
	Options DashboardVariableOptions `json:"options,omitempty"`
```

Reply:
Upon seeing all the above changes I realised there is a gap between tutone generation and manual modification. I will check if these manual changes need to be accommodated